coordinator/migrations/2023-09-28-163500_liquidity_options/up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE "liquidity_options" (
+       id SERIAL PRIMARY KEY NOT NULL,
+       rank SMALLINT NOT NULL,
+       title TEXT NOT NULL,
+       trade_up_to BIGINT NOT NULL,
```

Comment:
Might be worth adding `_sats` or a comment so that we know the unit of those fields.